### PR TITLE
gdub: init at 0.2.0

### PR DESCRIPTION
**Pull request by Larusso**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Larusso** [comment]
still important to me

**Comment by SuperSandro2000** [comment]
@Larusso Please fix the merge conflict.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4825,6 +4825,11 @@
     githubId = 55911173;
     name = "Gwendolyn Quasebarth";
   };
+  larusso = {
+    email = "manfred.endres@tslarusso.de";
+    github = "larusso";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4830]:
```suggestion
    github = "larusso";
    githubId = 2523575;
```

File: pkgs/development/tools/build-managers/gdub/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, makeWrapper, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "gdub";
+  version = "0.2.0";
+
+  nativeBuildInputs = [ makeWrapper ];
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/gdub/default.nix:7]:
```suggestion
```

File: pkgs/development/tools/build-managers/gdub/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    repo = "gdub";
+    rev = "v${version}";
+    sha256 = "0ddb5y5n02ajj7n07i8a6v833w67ls9adasvl9zwn73s98fg2w8j";
+  };
+
+  preferLocalBuild = true;
+  dontBuild = true;
+
+  phases = "installPhase";
+
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/gdub/default.nix:20]:
```suggestion

  nativeBuildInputs = [ makeWrapper ];

```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 82989` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>gdub</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -4825,6 +4825,12 @@
     githubId = 55911173;
     name = "Gwendolyn Quasebarth";
   };
+  larusso = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4828]: Please put this into a separate commit with the message ``maintainers: add larusso``.

File: pkgs/development/tools/build-managers/gdub/default.nix [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, makeWrapper, ... }:
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/gdub/default.nix:1]:
```suggestion
{ stdenv, fetchFromGitHub, makeWrapper }:
```

File: pkgs/development/tools/build-managers/gdub/default.nix [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    owner = "dougborg";
+    repo = "gdub";
+    rev = "v${version}";
+    sha256 = "0ddb5y5n02ajj7n07i8a6v833w67ls9adasvl9zwn73s98fg2w8j";
+  };
+
+  preferLocalBuild = true;
+  dontBuild = true;
+
+  phases = "installPhase";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/gdub/default.nix:17]:
```suggestion
```

File: pkgs/development/tools/build-managers/gdub/default.nix [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    outdir=$out/share/gdub/bin
+    mkdir -p $outdir
+    cp -r $src/bin/gw $outdir
+    chmod +x $outdir/gw
+    makeWrapper $outdir/gw $out/bin/gw \
```
Comment (SuperSandro2000) [inline pkgs/development/tools/build-managers/gdub/default.nix:26]: You can just use a symlink.

File: pkgs/top-level/all-packages.nix [hunk 8]
```diff
@@ -11459,6 +11459,7 @@ in
   gradle_4 = gradle_4_10;
   gradle_5 = res.gradleGen.gradle_5_6;
   gradle_6 = res.gradleGen.gradle_6_7;
+  gdub = callPackage ../development/tools/build-managers/gdub { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:11462]: Please sort this.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 82989` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gdub</li>
</ul>
</details>

**Comment by papojari** [comment]
https://www.gdub.rocks/ says that gdub is obsolete. Should it even be added anymore?

**Comment by Larusso** [comment]
@papojari I personally don't really care anymore. I had no time to upgrade this PR etc so it can now be closed.
